java-logging-spring/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-  id 'io.spring.dependency-management' version '1.0.6.RELEASE'
+  id 'io.spring.dependency-management' version '1.0.7.RELEASE'
```

Comment:
feels wrong the duplication but, you didn't introduce it...

Reply:
i know. just left in current structure